Makefile

```diff
@@ -186,17 +188,30 @@ lint-quick:
 	$(VIRTUAL_ENV)/bin/python manage.py makemigrations --dry-run --check --noinput || EXIT_STATUS=$$?; \
 	exit $${EXIT_STATUS}
 
-.PHONY: lint-fix
-lint-fix:
+.PHONY: lint-js-fix
+lint-js-fix:
 	EXIT_STATUS=0; \
-	npm run lint-fix ||  EXIT_STATUS=$$?; \
+	npm run lint-fix || EXIT_STATUS=$$?; \
+	exit $${EXIT_STATUS}
+
+# Use with caution, the automatic fixing might produce bad results
+.PHONY: lint-html-fix
+lint-html-fix:
+	EXIT_STATUS=0; \
+	$(VIRTUAL_ENV)/bin/djlint $(ARGUMENTS) --reformat --profile=django || EXIT_STATUS=$$?; \
+	exit $${EXIT_STATUS}
+
+.PHONY: lint-html-files
+lint-html-files:
+	EXIT_STATUS=0; \
+	$(VIRTUAL_ENV)/bin/djlint $(ARGUMENTS) --profile=django --ignore=H030,H031 || EXIT_STATUS=$$?; \
```

Comment:
added ignore for the following rules because they will produce an error every where and don't necessarily make sense or need feedback from PMs:
- H030 	Consider adding a meta description.
- H031 	Consider adding meta keywords.